ui/app/AppLayouts/Profile/views/wallet/AccountView.qml

```diff
@@ -1,158 +1,193 @@
+                Layout.fillWidth: true
+                Layout.preferredHeight: 1
+                color: Theme.palette.baseColor2
             }
-
-            InformationTile {
-                maxWidth: parent.width
-                primaryText: qsTr("Derivation Path")
-                secondaryText: root.account ? root.account.path : ""
-                visible: !!root.account && root.account.path
+            WalletAccountDetailsListItem {
+                id: derivationPath
+                Layout.fillWidth: true
+                title: qsTr("Derivation Path")
+                subTitle: root.account ? root.account.path : ""
+                visible: !!subTitle
             }
-
-            InformationTile {
-                maxWidth: parent.width
-                visible:root.account ? root.account.relatedAccounts.count > 0 : false
-                primaryText: qsTr("Related Accounts")
-                tagsModel: root.account ? root.account.relatedAccounts : []
-                tagsDelegate: StatusListItemTag {
-                    bgColor: Utils.getColorForId(model.colorId)
-                    bgRadius: 6
-                    height: 22
-                    closeButtonVisible: false
-                    asset.emoji: model.emoji
-                    asset.emojiSize: Emoji.size.verySmall
-                    asset.isLetterIdenticon: true
-                    title: model.name
-                    titleText.font.pixelSize: 12
-                    titleText.color: Theme.palette.indirectColor1
-                }
+            Separator {
+                Layout.fillWidth: true
+                Layout.preferredHeight: 1
+                color: Theme.palette.baseColor2
+                visible: derivationPath.visible
+            }
+            WalletAccountDetailsListItem {
+                Layout.fillWidth: true
+                title: qsTr("Stored")
+                subTitle: keyPair && keyPair.migratedToKeycard ? qsTr("on Keycard"): qsTr("On device")
             }
         }
+    }
+
+    Separator {
+        Layout.fillWidth: true
+        Layout.preferredHeight: 1
+        color: Theme.palette.baseColor2
+    }
 
+    RowLayout {
+        Layout.topMargin: 40
+        Layout.fillWidth: true
+        spacing: 8
         StatusButton {
+            Layout.fillWidth: true
+            text: keyPair && keyPair.migratedToKeycard? qsTr("Stop using Keycard") : qsTr("Migrate to Keycard")
+            onClicked: {
+                if (keyPair && keyPair.migratedToKeycard)
+                    console.warn("TODO: stop using Keycard")
+                else
+                    console.warn("TODO: move keys to a Keycard")
+            }
+        }
+        StatusButton {
+            Layout.fillWidth: true
             objectName: "deleteAccountButton"
-            visible: !!root.account && root.account.walletType !== "" 
-            text: qsTr("Remove from your profile")
+            visible: !!root.account && root.account.walletType !== ""
```

Comment:
`account.walletType` should never be `""`, so if you want to remove delete option for default wallet account you should check if `isWallet` is `true` instead (but ofc before that you should check if that role is exposed in this context, if not you will need to expose it).

Reply:
Aaah I dont know how I missed this, will do! I think this check should work 

walletType="generated" and path = "m/44'/60'/0'/0/0"
or
keyPair.type =Profile and path = "m/44'/60'/0'/0/0"